Makefile

```diff
@@ -9,13 +9,8 @@ default: install-build-deps install-gen-deps generate
 ci: testrace ## Test in continuous integration
 
 install-gen-deps: ## Install dependencies for code generation
-	# to avoid having to tidy our go deps, we `go get` our binaries from a temp
-	# dir. `go get` will change our deps and the following deps are not part of
-	# out code dependencies; so a go mod tidy will remove them again. `go
-	# install` seems to install the last tagged version and we want to install
-	# master.
-	@(cd $(TEMPDIR) && GO111MODULE=on go get github.com/mna/pigeon@master)
-	@(cd $(TEMPDIR) && GO111MODULE=on go get github.com/alvaroloes/enumer@master)
+	@go install github.com/mna/pigeon@v1.1.0
+	@go install github.com/alvaroloes/enumer@master
```

Comment:
why not pin this also?

Reply:
Ah let me leave a comment on that one. The latest release is outdated. So keeping it at master for now.

Reply:
does it allow for a @<commit-id>?

Reply:
I believe it does. But TBH a tag would be preferred. Seeing as it was set to master before, and the code has not changed, I will keep it as it is. There appears to be a few forks that have since been updated, which we may want to look into. 

I'm going to merge this change but please continue the conversation as we can iterate on the change.

Reply:
I'm generally worried about these loose dependencies, because they generally mean the build has less chances of being reproducible in the future. And being non-reproducible has a tendency to break, not only this upstream project, but also downstream clients. Downstream clients generally seem to a have a tendency to be "stuck" until upstream fixes these kind of problems, or they themselves try to fix it (if they even known how) with(out) upstream help, but until a fix is ready, someone might be "stuck".

Please note that I understand this is a fine balance between practicality and maintainability, and I'm not trying to push anything, I'm just expressing my opinion.

FWIW, I was a tiny bit impacted by this; mind you, by my own doing! I though I was pinning the go version, but I was not really doing that, so this was an enlightenment moment for me! and at the end, I was graceful that this has happened!